content/docs/examples/multicluster/index.md

```diff
@@ -0,0 +1,312 @@
+    $ export TELEMETRY_POD_IP=$(kubectl -n istio-system get pod -l istio-mixer-type=telemetry -o jsonpath='{.items[0].status.podIP}')
+    {{< /text >}}
+
+1.  Generate remote cluster manifest:
+
+    {{< text bash >}}
+    $ helm template install/kubernetes/helm/istio-remote --namespace istio-system \
+      --name istio-remote \
+      --set global.remotePilotAddress=${PILOT_POD_IP} \
+      --set global.remotePolicyAddress=${POLICY_POD_IP} \
+      --set global.remoteTelemetryAddress=${TELEMETRY_POD_IP} \
+      --set global.proxy.envoyStatsd.enabled=true \
+      --set global.proxy.envoyStatsd.host=${STATSD_POD_IP} > $HOME/istio-remote.yaml
+    {{< /text >}}
+
+## Install remote cluster manifest
+
+The following installs the minimal Istio components and enables automatic sidecar injection on
+the namespace `default` in the remote cluster:
+
+{{< text bash >}}
+$ kubectl config use-context "gke_${proj}_${zone}_cluster-2"
+$ kubectl create ns istio-system
+$ kubectl apply -f $HOME/istio-remote.yaml
+$ kubectl label namespace default istio-injection=enabled
+{{< /text >}}
+
+## Create remote cluster's kubeconfig for Istio Pilot
+
+The `istio-remote` helm chart creates a service account with minimal access for use by Istio Pilot
+discovery.
+
+1.  Prepare environment variables for building the `kubeconfig` file for the service account `istio-multi`:
+
+    {{< text bash >}}
+    $ export WORK_DIR=$(pwd)
+    $ CLUSTER_NAME=$(kubectl config view --minify=true -o "jsonpath={.clusters[].name}")
+    $ CLUSTER_NAME="${CLUSTER_NAME##*_}"
+    $ export KUBECFG_FILE=${WORK_DIR}/${CLUSTER_NAME}
+    $ SERVER=$(kubectl config view --minify=true -o "jsonpath={.clusters[].cluster.server}")
+    $ NAMESPACE=istio-system
+    $ SERVICE_ACCOUNT=istio-multi
+    $ SECRET_NAME=$(kubectl get sa ${SERVICE_ACCOUNT} -n ${NAMESPACE} -o jsonpath='{.secrets[].name}')
+    $ CA_DATA=$(kubectl get secret ${SECRET_NAME} -n ${NAMESPACE} -o "jsonpath={.data['ca\.crt']}")
+    $ TOKEN=$(kubectl get secret ${SECRET_NAME} -n ${NAMESPACE} -o "jsonpath={.data['token']}" | base64 --decode)
+    {{< /text >}}
+
+    __NOTE__: An alternative to `base64 --decode` is `openssl enc -d -base64 -A` on many systems.
+
+1.  Create a `kubeconfig` file in the working directory for the service account `istio-multi`:
+
+    {{< text bash >}}
+    $ cat <<EOF > ${KUBECFG_FILE}
+    apiVersion: v1
+    clusters:
+       - cluster:
+           certificate-authority-data: ${CA_DATA}
+           server: ${SERVER}
+         name: ${CLUSTER_NAME}
+    contexts:
+       - context:
+           cluster: ${CLUSTER_NAME}
+           user: ${CLUSTER_NAME}
+         name: ${CLUSTER_NAME}
+    current-context: ${CLUSTER_NAME}
+    kind: Config
+    preferences: {}
+    users:
+       - name: ${CLUSTER_NAME}
+         user:
+           token: ${TOKEN}
+    EOF
+    {{< /text >}}
+
+At this point, the remote clusters' `kubeconfig` files have been created in the `${WORK_DIR}` directory.
```

Comment:
Change to active voice:
At this point, you have created the `kubeconfig` files for the remote clusters in the `${WORK_DIR}` directory.